.github/workflows/test-android-device.yaml

```diff
@@ -43,7 +43,8 @@ jobs:
       - name: Set up Flutter
         uses: subosito/flutter-action@v2
         with:
-          channel: ${{ matrix.flutter-channel }}
+          channel: 'stable'
```

Comment:
channel defaults to `stable`
```suggestion
```

Reply:
It isn't said in action's readme. But let's remove it.